service/docs/source/admin.rst

```diff
@@ -24,11 +24,29 @@ command can be used to interact with the ``geopm`` Systemd Unit.
 GEOPM Service Files
 -------------------
 
-In addition to the files installed and maintained by the installation
-packages, the GEOPM service may create and modify files at runtime.
-These include files in ``/etc/geopm-service`` that control access
-settings and files in ``/run/geopm-service`` that track information
-about clients that are actively using the service.
+In addition to the files provided by the installation packages, the
+GEOPM service may create and modify files at run-time.  These include
+files in ``/etc/geopm-service`` that control access settings and files
+in ``/run/geopm-service`` that track information about clients that
+are actively using the service.
+
+All files and directories within ``/etc/geopm-service`` or
+``/run/geopm-service`` are created by the GEOPM Service with
+restricted access permissions and root ownership.  The GEOPM Service
+will not read any file or directory if they are modified to have more
+permissive access restrictions, non-root ownership, or if they are
+replaced by a symbolic link or other non-regular file.  If these
+checks fail, the file or directory will be renamed to include a UUID
+string and a warning is printed in the syslog.  These renamed files or
+directories enable an administrator to perform an investigation into
+problem, but they will not be used by the GEOPM Service in any way.
```

Comment:
Minor suggestion: perhaps this might be a good opportunity to reference the service security guide for further details about security measures/checks we have in place.